enclave-runner/src/loader.rs

```diff
@@ -260,6 +262,21 @@ impl<'a> EnclaveBuilder<'a> {
         self
     }
 
+    /// Set arguments passed to the main function in the enclave.
+    /// **NOTE:** This is not an appropriate channel for passing secrets or
+    /// security configurations to the enclave.
+    ///
+    /// **NOTE:** This is only applicable to [`Command`] enclaves.
+    /// Setting command arguments and then calling [`build_library`] will cause
+    /// a panic.
+    ///
+    /// [`Command`]: struct.Command.html
+    /// [`build_library`]: struct.EnclaveBuilder.html#method.build_library
+    pub fn command_arguments(&mut self, cmd_args: Vec<String>) -> &mut Self {
```

Comment:
See https://doc.rust-lang.org/std/process/struct.Command.html#method.args

And arguments should be of type `Vec<u8>`, not `String`